scope.go

```diff
@@ -34,6 +36,12 @@ var (
 	NoopScope, _ = NewRootScope(ScopeOptions{Reporter: NullStatsReporter}, 0)
 	// DefaultSeparator is the default separator used to join nested scopes
 	DefaultSeparator = "."
+	// ComponentSeparator is the separator used to separate a metric's name and it's tags in its ID
+	ComponentSeparator = "+"
```

Comment:
Wonder if we should make these unexported, they seem to be internal implementation details to the scope for computing unique IDs